app/Http/Controllers/User/ProfileController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Http\Controllers\User;
+
+use App\Http\Requests\ProfileRequest;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\View\View;
+
+class ProfileController extends Controller
+{
+    /**
+     * @return View
+     */
+    public function showEditProfileForm(): View
```

Comment:
нет смысла в таком названии.
```suggestion
    public function show(): View
```